assets/js/modules/thank-with-google/components/common/PositionRadio.js

```diff
@@ -26,38 +26,38 @@ import { useCallback } from '@wordpress/element';
 import Data from 'googlesitekit-data';
 import {
 	MODULES_THANK_WITH_GOOGLE,
-	BUTTON_PLACEMENT_STATIC_ABOVE_CONTENT,
-	BUTTON_PLACEMENT_STATIC_AUTO,
-	BUTTON_PLACEMENT_STATIC_BELOW_1ST_PARAGRAPH,
-	BUTTON_PLACEMENT_STATIC_BELOW_CONTENT,
+	CTA_PLACEMENT_STATIC_ABOVE_CONTENT,
+	CTA_PLACEMENT_STATIC_AUTO,
+	CTA_PLACEMENT_STATIC_BELOW_1ST_PARAGRAPH,
+	CTA_PLACEMENT_STATIC_BELOW_CONTENT,
 } from '../../datastore/constants';
 import Radio from '../../../../components/Radio';
 const { useSelect, useDispatch } = Data;
 
 export default function PositionRadio() {
-	const { setButtonPlacement } = useDispatch( MODULES_THANK_WITH_GOOGLE );
+	const { setCTAPlacement } = useDispatch( MODULES_THANK_WITH_GOOGLE );
 
-	const buttonPlacement = useSelect( ( select ) =>
-		select( MODULES_THANK_WITH_GOOGLE ).getButtonPlacement()
+	const ctaPlacement = useSelect( ( select ) =>
+		select( MODULES_THANK_WITH_GOOGLE ).getCTAPlacement()
 	);
 
 	const onChange = useCallback(
 		( { target } = {} ) => {
 			const { value: placement } = target || {};
-			setButtonPlacement( placement );
+			setCTAPlacement( placement );
 		},
-		[ setButtonPlacement ]
+		[ setCTAPlacement ]
 	);
 
 	return (
 		<div className="googlesitekit-twg-setting-field googlesitekit-twg-position-radio">
 			<h4>{ __( 'Position', 'google-site-kit' ) }</h4>
 			<div className="googlesitekit-twg-position-radio__option">
 				<Radio
-					id={ `button-placement-${ BUTTON_PLACEMENT_STATIC_AUTO }` }
+					id={ `button-placement-${ CTA_PLACEMENT_STATIC_AUTO }` }
```

Comment:
Don't we have to change `button-placement` to `cta-placement`?

Reply:
Yep, updated.